client/src/app/components/footer/footer.component.scss

```diff
@@ -0,0 +1,7 @@
+.container {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+/* Ignore this - here for Travis*/
```

Comment:
@OmerSchechter I saw travis works on your repo - great! Now, just remove this text (leave here one empty line at the end of the file).